flask_jwt_extended/tokens.py

```diff
@@ -76,7 +76,7 @@ def encode_refresh_token(identity, secret, algorithm, expires_delta, csrf):
     return _encode_jwt(token_data, expires_delta, secret, algorithm)
 
 
-def decode_jwt(encoded_token, secret, algorithm, csrf):
+def decode_jwt(encoded_token, secret, algorithm, csrf, identity):
```

Comment:
Could we add this to the `encode_refresh_token` and `encode_access_token` too? I think the `identity` kwarg is already in use, so we could use something like `identity_claim` or something instead? In fact, maybe we should change this to `identity_claim` as well?

We will need to update the calls to `encode_access_token`  and `encode_refresh_token` as well. I think those are only used in the `jwt_manager.py`.